Practical Example/RW_interview_prep2/RW_interview_prep/Model/DataModel.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  DataModel.swift
+//  RW-takeHomeProject
+//
+//  Created by Duc Dang on 8/25/20.
+//  Copyright © 2020 Duc Dang. All rights reserved.
+//
+
+import Foundation
+
+struct Arcdata : Decodable {
+    var data : [ArticleAndVideoData]
+}
+
+struct ArticleAndVideoData: Identifiable, Decodable, Comparable {
```

Comment:
I'm guessing your problem here is that this struct could represent a video or an article. Instead of using `and` in the name, think of a more abstract term, like `ItemData` or `PostData`. Furthermore, adding `Data` to the name is kind of redundant, it adds no information. So just `Post` or `Item` works just as well as your current name.